i18n/po/i18n_nl.po

```diff
@@ -22,6 +22,21 @@ msgstr ""
 msgid "Assertion violation processing RM schemas; original recipient: {0}"
 msgstr ""
 
+#: ../tools/src/main/java/com/nedap/archie/rmobjectvalidator/RMObjectValidationMessageIds.java:13
+#, java-format
+msgid "Attribute does not match cardinality {0}"
+msgstr "Attribuut moet {0} keer voorkomen"
+
+#: ../tools/src/main/java/com/nedap/archie/rmobjectvalidator/RMObjectValidationMessageIds.java:15
+#, java-format
+msgid "Attribute has {0} occurrences, but must be {1}"
+msgstr "Attribuut komt {0} keer voor, maar dit moet {1} zijn"
+
+#: ../tools/src/main/java/com/nedap/archie/rmobjectvalidator/RMObjectValidationMessageIds.java:14
+#, java-format
+msgid "Attribute {0} of class {1} does not match existence {2}"
+msgstr "Attribuut {0} van class {1} komt niet overeen met existence {2}"
```

Comment:
for existence, the only options are prohibited, mandatory or optional. So better might be to do 'attribute {0} of class {1} is prohibited, but present', or 'this attribute is mandatory, but missing'.

Reply:
As this is also problem with the English message, this out of scope of this pull request.